tests/plugins/stylescombo/manual/langattribute.html

```diff
@@ -0,0 +1,38 @@
+<div>
+	<div id="editor"></div>
+
+	<div>
+		<strong id="editorLang"></strong>
+		<div>Panel list attributes:</div>
+		<div id="result"></div>
+	</div>
+</div>
+
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		lang: 'en',
+		stylesSet: [
+			{ name: 'testName1', element: 'strong', language: 'pl' },
+			{ name: 'testName2', element: 'span' }
+		],
+		on: {
+			instanceReady: function( evt ) {
+				var editorLangElement = CKEDITOR.document.getById( 'editorLang' );
+				editorLangElement.appendHtml( 'Editor language: ' + evt.editor.langCode );
+			},
+			panelShow: function( evt ) {
+				var panelItems = evt.data._.panel
+					.getHolderElement()
+					.find( 'a' )
+					.toArray(),
+				resultElement = CKEDITOR.document.getById( 'result' );
+				resultElement.setHtml( '' );
+
+				CKEDITOR.tools.array.forEach( panelItems, function( element, index ) {
+					resultElement.appendHtml( 'Item ' + index + ': ' + JSON.stringify( element.getAttributes() ) + '</br></br>' );
```

Comment:
The dump of all attributes in the JSON format is quite unreadable. Maybe just display the value of the `[lang]` attribute?